src/altinn-app-frontend/package.json

```diff
@@ -84,6 +84,7 @@
     "react-styleguidist": "12.0.0",
     "redux-mock-store": "1.5.4",
     "redux-saga-test-plan": "4.0.6",
+    "resize-observer-polyfill": "1.5.1",
```

Comment:
Hvorfor trenger vi en polyfill? :thinking: Jeg trodde ikke vi trengte støtte for IE11: https://caniuse.com/resizeobserver

Reply:
Ahh, ok, brukt i testen, ser jeg. Regner med at node 16 ikke støtter den da.

Reply:
Det var testen som streiket for resize-observer ja. Av det jeg så da jeg undersøkte litt virket det som om jest ikke definerer en resize observer så den måtte mockes. Det som bruker resize-observer er den ene dependencien til radix-ui/popover som brukes i PopoverPanel komponenten i designsystemet.